sync/course-db.js

```diff
@@ -895,8 +895,8 @@ async function validateAssessment(assessment, questions) {
     };
     (assessment.zones || []).forEach(zone => {
         (zone.questions || []).map(zoneQuestion => {
-            if (!allowRealTimeGrading && Array.isArray(zoneQuestion.points)) {
-                errors.push(`Cannot specify an array of points for a question if real-time grading is disabled`);
+            if (!allowRealTimeGrading && Array.isArray(zoneQuestion.points) && zoneQuestion.points.length > 1) {
+                errors.push(`Cannot specify more than 1 point value for a question if real-time grading is disabled`);
```

Comment:
I'm slightly worried about the confusion between "1 point" meaning "this question is worth 1 point" and "1 point value" meaning "not an array with length more than one". Maybe it would be clearer to use:
```suggestion
                errors.push(`Cannot specify an array of multiple point values for a question if real-time grading is disabled`);
```